api-report/container-runtime-definitions.api.md

```diff
@@ -67,6 +68,14 @@ export interface IContainerRuntimeEvents extends IContainerRuntimeBaseEvents {
     (event: "localHelp", listener: (message: IHelpMessage) => void): any;
 }
 
+// @public @deprecated (undocumented)
+export interface IDataStoreWithBindToContext_Deprecated extends IDataStore {
```

Comment:
is there a need to extend data store here?

Reply:
Yeah I think it's correct this way.  The thing being cast to this is IDataStore, and Bohemia uses it as such (not just to call bindToContext).  https://dev.azure.com/office/OC/_git/office-bohemia?path=/packages/base-container/src/handlers/createComponentHandler.ts&version=GBmaster&line=316&lineEnd=330&lineStartColumn=1&lineEndColumn=4&lineStyle=plain&_a=contents